.github/workflows/run-tests.yml

```diff
@@ -0,0 +1,23 @@
+name: Run tests
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  publish-npm:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check-out the repository
+        uses: actions/checkout@v2
+      - name: Setup node version 16.13.0 (LTS)
+        uses: actions/setup-node@v2
+        with:
+          node-version: '16.13.0'
+          registry-url: 'https://registry.npmjs.org'
+      - run: npm ci
+      - run: npm run lint:fix
```

Comment:
Do we want to run `lint:fix` or only `lint`?

Reply:
Mmm, yeah. Maybe `lint:fix` is not worth it at this point 👍